mozanalysis/bootstrap.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+from functools import partial
+
+
+def bootstrap(iteration, stat_fn, broadcast_data):
+    np.random.seed()
```

Comment:
One more thing that I forgot about -- if you seed this with the iteration number (plus maybe some other characteristic of the original sample like, n), you can make this deterministic which would be nice for repeatability